### DISCUSSION TITLE
Error running on PythonAnywhere.

### DISCUSSION THREAD

**Discussion by anymeofu** [aahnik/tgcf#44] [Q&A] [answered]
First of all thanks for script.
iI have no issues running it on Desktop(Windows).

Recently I wanted to try On Pythonanywhere and I got this error. I hope You could Help !
![Capture](https://user-images.githubusercontent.com/76214556/112755694-79555500-8fff-11eb-9fd8-1a3571a9669a.PNG)

**Comment by aahnik** [comment, marked as answer]
The error that you have shown is not due to Python Anywhere.

You are most probably using python 2 (that's why showing syntax error).

Run `python --version`. It should be above 3.

Use the `python3` command for safety everywhere.

Like: `python3 forwarder.py`

---------
**Note:**

Python Anywhere free account does not allow connecting to certain servers like that of Telegram. There may be some workarounds to use this in the free tier. If I get to know them, I will surely post them.

**Reply by anymeofu** [reply]
Thank you !
